### ISSUE TITLE
Project Check #7

### ISSUE THREAD

**Issue by PaarasPurohit** [PaarasPurohit/apcompsciportfolio#26]
- [x] [Complete popcorn hacks in the tech talks](https://paaraspurohit.github.io/apcompsciportfolio/sorting-algorithm-hacks) +1 pts
- [x] Incorporating 2D and animation into our project
- [Implementing Gravity](https://tanayp327.github.io/PhysicsArcade/physics)
- [Implementing Movement](https://tanayp327.github.io/PhysicsArcade/movement)
- [Ederick's Work](https://tanayp327.github.io/PhysicsArcade/snake)

About my work:

This week was more productive than before. Not only did I get movement working, but I also found how to implement gravity in our games. Words cannot describe how extremely important this mechanic is, as we are doing our project on teaching mechanical physics and focusing on projectile motion as our audience hook. The whole week, I was faced with many errors, and half the time I did not know what was causing them. However, using pair programming, documentation, and admittedly, some ChatGPT, I was able to resolve the issues and get tangibles working as linked to the ticket. We also are decreasing the amount of goofing off that we do in class, which is good for our team productivity, considering how big of a group we are, and shows our acknowledgement of errors and fixing them. 

tl;dr: I got extremely important tangibles working and our team productivity exponentially increased.

(Chinmay, Tanay, Raunak, and Qais never sent in their review tickets, although they should most likely send them via Slack.)
